### ISSUE TITLE
Css Properties Copy Paste Issue

### ISSUE THREAD

**Issue by vijayhardaha** [brave/brave-browser#19505]
When I inspect elements and try to copy-paste the CSS properties, px, rem, em properties wrap in a new line for some reason.

I have recorded the video of the issue please have a look. This is not a website issue. I have tested in safari, it works fine. I think it's a brave issue.

[https://www.dropbox.com/s/6jx0ema2qepvvwe/CSS%20properties%20copy%20paste%20issue.mov?dl=0](https://www.dropbox.com/s/6jx0ema2qepvvwe/CSS%20properties%20copy%20paste%20issue.mov?dl=0)

**Comment by rebron** [comment]
@vijayhardaha If you have Chrome installed, are you seeing this reproduce in Chrome?

**Comment by vijayhardaha** [comment]
Chrome has no issue, just checked.
Also tested on incognito mode with no extension but brave still has that issue.

Video link [https://www.dropbox.com/s/sx2iutctbo3m8va/Chrome%20is%20Fine-%20Brave%20has%20issue.mov?dl=0](https://www.dropbox.com/s/sx2iutctbo3m8va/Chrome%20is%20Fine-%20Brave%20has%20issue.mov?dl=0)

**Comment by emmanuelvlad** [comment]
It's very annoying and impossible to work with

**Comment by milkbottlelough** [comment]
I too am seeing this issue, it seems to be related to the feature of being able to select the units in increments using the mouse etc.  Users should be able to toggle this feature; I doubt it will be used all that much anyway as most front-end devs (in my experience and including myself) tend to input the unit value by hand or by using the arrow keys.

![image](https://user-images.githubusercontent.com/45833805/146380211-1abb9a00-8983-4210-94c1-65a608a33666.png)

**Video:**

https://user-images.githubusercontent.com/45833805/146380814-a79dd420-6ede-497c-8bbc-b5d65d2d69f9.mp4

**Device**
Model Name:	MacBook Pro
Processor Name:	Quad-Core Intel Core i5
MacOS Catalina (10.15.7)

**Brave**
Version 1.32.115 Chromium: 96.0.4664.93 (Official Build) (x86_64)

**Comment by vijayhardaha** [comment]
Yeah, I am not liking the feature of increment value by mouse drag, this should have option somewhere in developer tools to turn on/off this feature, people generally like to click and type the value by hand or use up down arrow keys to increase decrease the values or choose the next prev values as like it was before.

**Comment by vijayhardaha** [comment]
# Additional information on this issue.

### When copy-pasting on VSCODE or in a text file.
```
.comment-body {
    width: 100%;
    padding: 15
px
;
    overflow: visible;
    font-size: 14px;
}
```

### When copy-pasting on MacOS notes app.
```
.comment-body {
    width: 100%;
    padding: 15pxcmmminpcptchemremvhvwvminvmax;
    overflow: visible;
    font-size: 14px;
}
```

> macOS 12.1
> Brave is up to date
> Version 1.33.106 Chromium: 96.0.4664.110 (Official Build) (arm64)

**Comment by Tomgroot** [comment]
Same problem here...

Copying this:

![image](https://user-images.githubusercontent.com/4732713/147218114-45f2f555-9555-44ba-9ce6-add75546bc66.png)

Results in this inside PHPStorm:

![image](https://user-images.githubusercontent.com/4732713/147218324-078a6937-69b0-4b44-8c6b-164fb8efdfc7.png)

**Comment by milkbottlelough** [comment]
Thanks @Tomgroot for the extra info.

Just tried the nightly build and this issue still persists.
